packages/lib/bookings/filterHostsByLeadThreshold.ts

```diff
@@ -105,7 +107,11 @@ export const filterHostsByLeadThreshold = async <T extends BaseHost<BaseUser>>({
   if (maxLeadThreshold === 0) {
     throw new Error(errorCodes.MAX_LEAD_THRESHOLD_FALSY);
   }
-  if (maxLeadThreshold === null || hosts.length < 1) {
+  if (
+    maxLeadThreshold === null ||
+    hosts.length < 1 ||
+    (eventType.team?.rrTimestampBasis && eventType.team.rrTimestampBasis !== RRTimestampBasis.CREATED_AT)
```

Comment:
Load balancing doesn't work because it depends on the selected time slot, which we don't know yet